test/y2storage/proposal_scenarios_x86_test.rb

```diff
@@ -133,7 +133,7 @@
     end
 
     context "in a PC with an empty partition table" do
-      let(:scenario) { "empty_hard_disk_mbr_50GiB" }
+      let(:scenario) { "empty_hard_disk_gpt_50GiB" }
```

Comment:
This in fact changes the starting point, which looks a bit like cheating. What about this?

```ruby
let(:scenario) { "empty_hard_disk_mbr_50GiB" }
let(:expected_scenario) { "empty_hard_disk_gpt_50GiB" }
```

That would indeed prove it behaves as we want. With your change we are actually doing the same tests twice (there is another "_in a PC with an empty GPT partition table_" below in the same file).

Reply:
Yes, that was I thought, but then I do it wrong.